graphtool.js

```diff
@@ -1533,11 +1533,25 @@ d3.json(typeof PHONE_BOOK !== "undefined" ? PHONE_BOOK
     }
     let phoneSel = doc.select("#phones").selectAll()
         .data(allPhones).join("div")
-    .attr("class","phone-item")
+        .attr("class","phone-item")
         .on("mouseover", bg(true, p => getDivColor(p.id===undefined?nextPhoneNumber():p.id, true)))
         .on("mouseout" , bg(false,p => p.id!==undefined?getDivColor(p.id,p.active):null))
         .call(setClicks(showPhone));
     phoneSel.append("span").text(p=>p.fullName);
+    
+    //Adding the selection button
+    //TODO: This breaks the whole selection stuff, now the graph tool can only add
+    
```

Comment:
Changes break existing functionality?

Reply:
Should not happen anymore, I forgot to delete this TO-DO. This has been solved